store/beestore/beestore_test.go

```diff
@@ -16,13 +18,21 @@ import (
 	"github.com/ethersphere/bee/v2/pkg/swarm"
 	"github.com/gorilla/websocket"
 	"github.com/uncloud-registry/swarm-driver/store/beestore"
-	"github.com/uncloud-registry/swarm-driver/store/teststore"
 )
 
 func TestStoreCorrectness(t *testing.T) {
-	tstore := teststore.NewSwarmInMemoryStore()
-	srvURLStr := newTestServer(t, tstore)
 
+	batchID, err := getNewTestBatchID()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	bstore, err := beestore.NewBeeStore("localhost", 1633, false, batchID, false, true)
+	if err != nil {
+		return
+	}
+	fmt.Println("Beestore: ", bstore)
+	srvURLStr := newTestServer(t, bstore)
```

Comment:
The server is simulating a swarm node. Which is why it uses the inmem store. Beestore is for the client side. So this will not work only. The swarm node only has to store things locally for the test.